integration_test.go

```diff
@@ -34,6 +35,8 @@ import (
 	"cloud.google.com/go/spanner/admin/database/apiv1/databasepb"
 	instance "cloud.google.com/go/spanner/admin/instance/apiv1"
 	"cloud.google.com/go/spanner/admin/instance/apiv1/instancepb"
+
+	"github.com/googleapis/go-gorm/testutil"
```

Comment:
I see that the module is named as github.com/googleapis/go-gorm.
But shouldn't this be github.com/googleapis/go-gorm-spanner as this implementation is spanner specific?